README.md

```diff
@@ -35,12 +35,12 @@ To enable the provider, create or edit the file at `./config/plugins.js`.
 This is an example `plugins.js` file for Azure storage:
 
 ```js
-
 module.exports = ({ env }) => ({
   upload: {
     config: {
       provider: "strapi-provider-upload-azure-storage",
       providerOptions: {
+        auth_type: env("STORAGE_AUTH_TYPE", "default"),
```

Comment:
why wihout this line produces error ? did msoft change something?

Reply:
@Achmadrendra1 You need to include an `authType` now. It's so to distinguish between the two key types. If you're using the original variables, all you need to do it include that line.

Reply:
Sorry. In hindsight I should have published this as a major version since that was a breaking change.